target/linux/ipq40xx/Makefile

```diff
@@ -3,7 +3,7 @@ include $(TOPDIR)/rules.mk
 ARCH:=arm
 BOARD:=ipq40xx
 BOARDNAME:=Qualcomm Atheros IPQ40XX
-FEATURES:=squashfs fpu ramdisk nand
```

Comment:
Hm, I've read the relevant commit message text... and I think you could make a case for a separate sub-target for that. I've done the same with the APM821xx / WD My Book Live. It lives there in a "sata" subtarget with boot-part and rootfs-part.

This would also allow you to "#ifdef" around that problematic SDI revert patch you have... You could make add a "Kconfig Symbol" that you only enable for the sub-target...

Reply:
Thanks. This was actually a key question I was hoping to get feedback on, and I didn't get any with the mailing list posting.

I'll look at doing a subtarget. I agree that makes sense.

Reply:
hmm, I just remembered. There's also `of_machine_is_compatible("$machine_compatible")`. so, you could have the patches and skip the sub-target. hm

Reply:
I've pushed a subtarget to address other concerns (additional USB drivers).